UnitTests/tst_deploytest.cpp

```diff
@@ -1517,6 +1518,23 @@ void deploytest::testDisableRunScripts() {
                    "-disableRunScript", "TestOnlyC"}, &comapareTreeqif);
 }
 
+void deploytest::testQifOut() {
+    TestUtils utils;
+
+#ifdef Q_OS_UNIX
+    QString bin = TestBinDir + "TestOnlyC";
+#else
+    QString bin = TestBinDir + "TestOnlyC.exe";
+#endif
+
+    auto result = utils.createTree({{DISTRO_DIR + "/QIF_OUT.exe"},
+                                   {DISTRO_DIR + "/QIF_OUT.exe.md5"}});
+
+    // Run deploy installer
+    runTestParams({"-bin", bin, "clear",
+                   "qif", "-qifOut", "QIF_OUT.exe"}, &result);
```

Comment:
Hmm, this test looks wired.

You said "on Windows, the exe suffix will be added to the installer automatically". However, in this test, you are using `"-qifOut", "QIF_OUT.exe"` and the expected output is `DISTRO_DIR + "/QIF_OUT.exe"` . This test passed on both Linux and Windows, does it mean that the suffix will be processed smartly? I mean, if you provided a file name with a ".exe" suffix in "-qifOut", no more suffix will be added no matter what type of operating system is, am I right? If so, what about ".run" suffix?

Reply:
no, the binarycreator tool  automaticity added only one suffix - it is exe for windows and only when output name do not contains exe suffix.

Reply:
@EndrII If I specified a wrong suffix such as .txt, would the .exe suffix be added automatically?

Reply:
i think  yes